src/layout/layout.js

```diff
@@ -140,6 +139,12 @@
     return window.matchMedia(query);
   };
 
+  var mediaQuery = MaterialLayout.prototype.matchMedia_(
+    /** @type {string} */ (MaterialLayout.prototype.Constant_.MAX_WIDTH));
+  mediaQuery.onchange = screenSizeHandler;
+
+  MaterialLayout.prototype.screenSizeMediaQuery_ = mediaQuery;
```

Comment:
Why are you using `prototype`? This appears to be used as a static field, so it should just be on `MaterialLayout` directly. And what's the point of having both a `var` and a property?

Reply:
Mostly because I'm new to ES5 JS. I figured out what you mean though.